variables.tf

```diff
@@ -31,6 +31,12 @@ variable "sns_topic_arn" {
   description = "The SNS topic ARN to notify."
 }
 
+variable "create_alert_resources" {
```

Comment:
And the topic is required. How will we use the module in that case?

And maybe something like `alerts_enabled` would be more consistent naming. Please check other variables.